docs/hub-deployment-guide/upgrade-cluster/aws.md

```diff
@@ -0,0 +1,208 @@
+   time. So, update the eksctl config's version field one minor version.
+
+   Then, perform the upgrade which typically takes ~10 minutes.
+
+   ```bash
+   eksctl upgrade cluster --config-file=eksctl-config.yaml --approve
+   ```
+
+   ```{note}
+   If you see the error `Error: the server has asked for the client to provide credentials` don't worry, if you try it again you will find that the cluster is now upgraded.
+   ```
+
+4. *Upgrade node pools up to two minor versions above the k8s control plane*
+
+   Its documented that a node pools's k8s software components (`kubelet`) should
+   support being two minor versions out of sync from the control plane, so
+   consider if and what node pools you upgrade to avoid doing more operations
+   than needed.
+
+   If you upgrade from k8s 1.21 to 1.24, you can for example upgrade the k8s
+   control plane from 1.21 to 1.22, then upgrade the node pools from 1.21 to
+   1.24, followed by upgrading the control plane two steps in a row.
+
+   To upgrade (unmanaged) node pools, you delete them and then them back. When
+   adding them back, make sure your cluster config's k8s version is what you
+   want the node pools to be added back as.
+
+   1. Update the k8s version in the config temporarily
+
+      This is to influence the k8s software version for the nodegroup's we
+      create only. We can choose something two minor versions of the current k8s
+      control plane version.
+
+   2. Add a new core node pool (like `core-b`)
+
+      Rename (part 1/3) the config file's entry for the core node pool
+      temporarily when running this command, either from `core-a` to `core-b` or
+      the other way around.
+
+      ```bash
+      eksctl create nodegroup --config-file=eksctl-config.yaml --include="core-b" --install-nvidia-plugin=false
+      ```
+
+   3. Delete all old node pools (like `core-a,nb-*,dask-*`)
+
+      Rename (part 2/3) the core node pool again in the config to its previous
+      name, so the old node pool can be deleted with the following command.
+
+      ```bash
+      eksctl delete nodegroup --config-file=eksctl-config.yaml --include="core-b,nb-*,dask-*" --approve --drain=false
+      ```
+
+      Rename (part 3/3) the core node pool one final time in the config to its
+      new name, as that represents the state of the EKS cluster.
+
+   4. Re-create all non-core node pools (like `nb-*,dask-*`)
+
+      ```bash
+      eksctl create nodegroup --config-file=eksctl-config.yaml --include="nb-*,dask-*" --install-nvidia-plugin=false
+      ```
+
+   5. Restore the k8s version in the config
+
+      We adjusted the k8s version in the config to influence the desired version
+      of our created nodegroups. Let's restore it to what the k8s control plane
+      currently have.
+
+5. *Upgrad EKS add-ons (takes ~3*5s)*
+
+   ```bash
+   # upgrade the kube-proxy daemonset
+   eksctl utils update-kube-proxy --config-file=eksctl-config.yaml --approve
```

Comment:
Is this specific to any particular set of versions, or required for all? A link to where these commands are sourced from would also be helpful